### ISSUE TITLE
InvalidOutputType("Expected single element, got an empty list")

### ISSUE THREAD

**Issue by sergeyT2** [tomusdrw/rust-web3#655]
Hey everyone,

referencing this, obiously [https://github.com/tomusdrw/rust-web3/issues/115](url)

The contract I'm querying doesn't have explicit return value. Therefore I'm getting an error of 
`InvalidOutputType("Expected single element, got a list: []")`

How do I type it correctly? web3::contract::Result of something?

**Comment by tomusdrw** [comment]
The no-value return type is `()`, have you tried that?

**Comment by sergeyT2** [comment]
> The no-value return type is `()`, have you tried that?

Yeah, it says trait web3::contract::tokens::Tokenizable not implemented for `()`

**Comment by sergeyT2** [comment]
@tomusdrw anything happening?

**Comment by tomusdrw** [comment]
Do you mind providing a full example? I don't think parsing an empty value into an empty list as proposed in #656 is the correct behavior.
